shared/Views/Shared/Components/CourseDetails/_Advice.cshtml

```diff
@@ -6,3 +6,6 @@
 
 <h4 class="govuk-heading-m">Website support</h4>
 <p class="govuk-body">If you have feedback or have had a problem using Find postgraduate teacher training you can <a href="mailto:becomingateacher@digital.education.gov.uk">contact us by email</a>.</p>
+
+<h4 class="govuk-heading-m">Is there something wrong with this page?</h4>
```

Comment:
Should this be a child of "Website support", eg h5 govuk-heading-s?

Reply:
It's already a child of Advice and Support:

<img width="720" alt="screenshot 2018-09-28 at 14 51 25" src="https://user-images.githubusercontent.com/1650875/46212498-008db900-c32e-11e8-9184-681933fe0411.png">

It could well be, I think either hierarchy can make sense.